### PR TITLE
api-concepts.md: fix TypeMeta definition

### PR DESCRIPTION
**Pull request by loskutov**
Fix TypeMeta definition to match one in generated protobufs

### CODE REVIEW

**Comment by loskutov** [comment]
Eventually, the message is actually from https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/generated.proto, and it corresponds with the one in the documentation. It could be useful to have the link, though.

File: content/en/docs/reference/using-api/api-concepts.md [hunk 1]
```diff
@@ -704,6 +704,8 @@ An encoded Protobuf message with the following IDL:
   }
 ```
 
+The messages described above can also be found at https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/generated.proto.
```
Comment (Ritikaa96) [inline content/en/docs/reference/using-api/api-concepts.md:707]:
```suggestion
The messages described above can also be found at https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/generated.proto/.
```

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/using-api/api-concepts.md (hunk 1 above)
Comment (Ritikaa96) [reply content/en/docs/reference/using-api/api-concepts.md:707]: Maybe we can add it as a hyperlink and not just paste the link inline.
Something along the lines of :
```
The message above can also be found in kubernetes [sourcecode](https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/generated.proto/).
```

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

**Comment by tengqm** [comment]
@loskutov Mind revise the pointer to a Markdown link?

**Comment by tengqm** [comment]
Also, `master` is a moving target. Maybe we should point to a tag instead?

**Comment by divya-mohan0209** [comment]
Hello @loskutov , gentle reminder about this PR. Would you be able to address the changes suggested by @tengqm and @Ritikaa96 ?

**Comment by Ritikaa96** [comment]
Sure @divya-mohan0209 . 

> Also, `master` is a moving target. Maybe we should point to a tag instead?

Am not able to understand what is requested here. a bit confused . should we change the `master` to branch name there?

**Comment by sftim** [comment]
I'd use a tag; what I'd use is `release-{{< skew currentVersion >}}`.

Why? Because it's useful to link to the version of that file for the relevant release. There's a risk that the link breaks, but that breakage will be obvious to any reader, whereas a link to stale code might not be.

Link checkers should be able to pick up a putative broken link, too. Right now we don't run automated link checking that I know of, but it's a well known technique.

**Comment by kbhawkey** [comment]
Hello @loskutov .
Can you address the suggestions or do you have questions?
If you are no longer actively working on your pull request, you can close it and reopen it at a later time.
Thanks

**Comment by divya-mohan0209** [comment]
Hello @loskutov , thank you for your contribution. However, since there has been no response from your side, we will be closing this PR. If you wish to continue work on this PR, please feel free to reopen it.
/close

**Pull request closed without merging**